gpt.py

```
@@ -1427,11 +1689,19 @@ def _outlook_fetch_otp_imap(email_addr: str, client_id: str, refresh_token: str,
     return ""
 
 
-def _outlook_fetch_otp(email_addr: str, client_id: str, refresh_token: str,
-                       known_ids: set = None, proxies: Any = None, timeout: int = 120) -> str:
+def _outlook_fetch_otp(
+    email_addr: str,
+    client_id: str,
+    refresh_token: str,
+    known_ids: set = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)